meta/runtime.yml

```diff
@@ -1,2 +1,2 @@
 ---
-requires_ansible: ">=2.11"
```

Comment:
Is this intended?

Reply:
Yes, This file is needed. However i am still testing this with 2.9 onwards.
So far i have made it work just upto 2.11+.